src/book/entities/Account.ts

```diff
@@ -136,6 +141,77 @@ export default class Account extends BaseEntity {
     // Placeholders are hierarchical accounts that don't containg
     // transactions, they are just parents of other accounts
     placeholder!: boolean;
+
+  async save(options?: SaveOptions): Promise<this> {
+    const account = await super.save(options);
+
+    const queryClient = Account.getRepository().manager.connection.options.extra?.queryClient;
+    if (queryClient) {
+      updateCache({ queryClient, account: this });
+    }
```

Comment:
fail gracefully for now but at some point we probably want to make this mandatory as without a queryClient, the frontend does not work